src/main.js

```diff
@@ -59,9 +75,9 @@ async function writeHTML(data, filename) {
     content.forEach((line) => {
       if(linecount ===0) {
         title = line;
-        dataForBody += `<h1>${line}</h1>\n`;
+        dataForBody += `<h1>${parseMarkdown(line)}</h1>\n`;

```

Comment:
In the code there is a function on line 113 called readFile, which is called from line 32 and passed the name from the file object. What I would do is I would check in the checkExists what the file type is, and then only if the type is md to parseMarkdown. As of now, ALL input files which exist are going through the parseMarkdown and this takes up resources, especially when none of the files are markdown.

Reply:
The object that gets passed into the createHTML function is of: 
{ 
name: 
type: 
}

Reply:
Thanks for the review.
I have updated my request.
Please check it out

Reply:
LGTM! Just needed to do one more function for the page title (in the tab). Thank you!